app/config/errors.php

```diff
@@ -519,6 +546,46 @@
         'description' => 'The project key has expired. Please generate a new key using the Appwrite console.',
         'code' => 401,
     ],
+    Exception::ROUTER_INVALID_TYPE => [
+        'name' => Exception::ROUTER_INVALID_TYPE,
+        'description' => 'Invalid domain configuration. Route type is not supported.',
+        'code' => 400,
+    ],
+    Exception::ROUTER_UNKNOWN_HOST => [
```

Comment:
General note, we might want to throw an HTML error here like we do for OAuth. I think it should be more descriptive, maybe with a link to the docs.

Reply:
We do. This error is hhrown only by Appwroute Router. It's configured to respond with error.phtml